src/librustc_typeck/diagnostics.rs

```diff
@@ -1972,6 +1972,88 @@ To learn more about traits, take a look at the Book:
 https://doc.rust-lang.org/book/traits.html
 "##,
 
+E0174: r##"
+This error occurs because of unboxed closure explicitly methods calls is
+experimental in current Rust version.
+
+Example of erroneous code:
+
+```compile_fail
+ fn foo<F: Fn(&str)>(mut f: F) {
+     f.call(("call",));
+     //~ ERROR explicit use of unboxed closure method `call`
```

Comment:
I got those examples from here:
https://github.com/rust-lang/rust/blob/master/src/test/compile-fail/feature-gate-unboxed-closures-method-calls.rs#L14

I'm going to change